srcpkgs/mercurial/template

```diff
@@ -1,23 +1,37 @@
 # Template file for 'mercurial'
 pkgname=mercurial
-version=5.6.1
+version=5.7.1
 revision=1
 build_style=python3-module
-hostmakedepends="python3 python3-setuptools python3-devel"
+hostmakedepends="python3 python3-setuptools python3-devel gettext"
 makedepends="python3-devel"
 depends="python3 ca-certificates"
-checkdepends="iana-etc tar unzip which xz"
+# cvs tests fail
+checkdepends="iana-etc tar unzip which xz subversion-python python3-docutils gnupg2
+ python3-Pygments"
 short_desc="Fast, lightweight source control management system"
 maintainer="Johannes <johannes.brechtmann@gmail.com>"
 license="GPL-2.0-or-later"
 homepage="https://www.mercurial-scm.org/"
 distfiles="https://www.mercurial-scm.org/release/mercurial-${version}.tar.gz"
-checksum=e55c254f4904c45226a106780e57f4279aee03368f6ff6a981d5d2a38243ffad
+checksum=cb5139144ccb2ef648f36963c8606d47dea1cb0e22aa2c055d6f860ce3fde7b0
+
+pre_check() {
+	if [ "$XBPS_TARGET_LIBC" = musl ]; then
+		# fix hardcoded strerror() value from glibc
+		vsed -e 's/Address already in use/Address in use/' -i tests/common-pattern.py
+	fi
+}
 
 do_check() {
-	cd tests
-	rm test-doctest.py test-hghave.t test-https.t test-merge-tools.t test-persistent-nodemap.t
-	python3 run-tests.py ${makejobs}
+	if [ "$XBPS_CHECK_PKGS" = full ]; then
+		_additional_test=--allow-slow-tests
+	fi
```

Comment:
else `_additional_test='test-s*'`, then please try to run tests localy, it is much faster